packages/angular-generator/src/expressions/decorator.ts

```diff
@@ -96,10 +96,13 @@ export class Decorator extends BaseDecorator {
         if (templates?.length) template += templates.join('');
         const slots = compileSlots(options);
         if (slots?.length) template += slots.join('');
+        const isInnerComp = ((this.expression.argumentsArray[0] as ObjectLiteral)
+          ?.properties.find((e) => e.key?.toString() === 'jQuery')?.value as ObjectLiteral)
+          ?.properties.find((e) => e.key?.toString() === 'register' && e.value?.toString() === 'true');
```

Comment:
"this.expression.arguments[0] as ObjectLiteral" can be replaced with "parameters"
maybe there are some other ways to shorten the expression